presto-docs/src/main/sphinx/release/release-0.240.rst

```diff
@@ -0,0 +1,73 @@
+=============
+Release 0.240
+=============
+
+**Highlights**
+==============
+* Add support for inlining SQL functions at query planning time.
+
+**Details**
+==============
+
+General Changes
+_______________
+* Fix compiler error due to incorrect LambdaDefinitionExpression canonicalization.
+* Fix compiler error in certain situations where sql functions with same lambda are used multiple times.
+* Add `IF EXISTS `and `IF NOT EXISTS` syntax to `ALTER TABLE`.
```

Comment:
Use double back quotes, and fix misplace whitespace.

Reply:
done. corrected in a couple of other locations as well.